cobalt/android/apk/app/src/main/java/dev/cobalt/coat/CobaltActivity.java

```diff
@@ -141,7 +122,7 @@ protected StarboardBridge getStarboardBridge() {
   @Override
   protected void onStart() {
     if (!isReleaseBuild()) {
-      getStarboardBridge().getAudioOutputManager().dumpAllOutputDevices();
+      // getStarboardBridge().getAudioOutputManager().dumpAllOutputDevices();
```

Comment:
Just a note that this and similar commented-out statements should have
a TODO(cobalt) or some other easy-to-grep statement, so they can be
tracked.